MVVM Module/Source/Feed Presentation/FeedViewModel.swift

```diff
@@ -20,14 +20,20 @@ final class FeedViewModel {
 
 	var onLoadingStateChange: Observer<Bool>?
 	var onFeedLoad: Observer<[FeedImage]>?
+	var onFeedLoadError: Observer<String?>?
```

Comment:
Suggestion: you might want to consider renaming `onFeedLoadError` to `onErrorStateChange`, since it can now be used to represent any errors (not only those related to `Feed`), but also no errors. So the message does not necessarily represent a `FeedLoadError`.
```suggestion
	var onErrorStateChange: Observer<String?>?
```

Reply:
done